server/lib/alert-manager.js

```diff
@@ -105,9 +118,11 @@ ${$t('server.email.signature')}
                     if (!alert.testBalance(balance))
                         continue;
 
+                    // Set the currency formatter
+                    let curr = account.currency || defaultCurrency;
+                    let currencyFormatter = currency.makeFormat(curr);
                     let text =
-                        alert.formatAccountMessage(account.title, balance);
-
+                        alert.formatAccountMessage(account.title, balance, currencyFormatter);
```

Comment:
Now that the limit is to 100 chars, can this fit on the above line, next to `text =`?